app/src/main/res/values/strings.xml

```diff
@@ -240,5 +240,11 @@
   <string name="administrator_controls_app_version">App Version</string>
   <string name="administrator_controls_account_actions_label">Account Actions</string>
   <string name="administrator_controls_log_out">Log Out</string>
+  <string name="log_out_dialog_cancel_button"><![CDATA[CANCEL]]></string>
+  <string name="log_out_dialog_okay_button">OK</string>
```

Comment:
It should be `Ok`. Theme we will take care of CAPS.

Reply:
I did as per suggestion but then a warning came up showing that "Ok is usually capitalized as OK" as I kept it as it is .

Reply:
Yea, I know that warning is valid also, but this gives us less control over theme. Therefore we prefer `Ok` instead of `OK`.